src/data_model/eng2math.jl

```diff
@@ -457,6 +457,8 @@ function _map_eng2math_switch!(data_math::Dict{String,<:Any}, data_eng::Dict{<:A
                 # "f_bus" => bus_obj["bus_i"],  # TODO enable real switches
                 "f_bus" => data_math["bus_lookup"][eng_obj["f_bus"]],
                 "t_bus" => data_math["bus_lookup"][eng_obj["t_bus"]],
+                "f_connections" => eng_obj["t_connections"],  # the virtual branch connects to the switch on the to-side
```

Comment:
This should be 
```"f_connections" => eng_obj["f_connections"]```,
as the branch replaces the entire switch now (until you actually add in the virtual bus and switch, then this can be correct depending on how you connect the switch to the virtual bus)